apps/portal/examples/postgrest-template/README.md

```
@@ -0,0 +1,639 @@
+CREATE ROLE anon NOLOGIN;
+CREATE ROLE authenticated NOLOGIN;
+
+-- Public endpoints (unauthenticated)
+GRANT SELECT ON public_posts TO anon;
+
+-- Private endpoints (authenticated users only)
+GRANT SELECT ON user_profile TO authenticated;
+GRANT INSERT, UPDATE ON user_profile TO authenticated;
+```
+
+### Custom Docker Image
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)